python/ray/data/read_api.py

```diff
@@ -1755,6 +1761,109 @@ def create_connection():
     )
 
 
+@PublicAPI(stability="alpha")
+def read_databricks_uc_tables(
+    *,
+    warehouse_id: str,
+    table: Optional[str] = None,
+    query: Optional[str] = None,
+    catalog: Optional[str] = None,
+    schema: Optional[str] = None,
+    parallelism: int = -1,
+    ray_remote_args: Optional[Dict[str, Any]] = None,
+) -> Dataset:
+    """
+    Read from a Databricks UC table or Databricks SQL execution result that queries
```

Comment:
Explicitly spell out UC since it's the first time this acronym is referenced?

Reply:
unity catalog :)